docs/_source/guides/llms/practical_guides/fine_tune.md

```diff
@@ -446,6 +446,12 @@ trainer = ArgillaTrainer(
 trainer.train(output_dir="sft_model")
 ```
 
+```{note}
+You can also initialize the `ArgillaTrainer` with an already initialized `model` and `tokenizer` for additional fine-grained control. This might be useful if you wish to ensure that the tokenizer adds the EOS token.
```

Comment:
maybe add a hint
```suggestion
You can also initialize the `ArgillaTrainer` with an already initialized `model` and `tokenizer` for additional fine-grained control. This might be useful if you wish to ensure that the tokenizer adds the EOS token. A lack of this token might result in the model generating endlessly.
```